pkgs/top-level/all-packages.nix

```diff
@@ -30106,7 +30106,9 @@ in
 
   faust2lv2 = callPackage ../applications/audio/faust/faust2lv2.nix { };
 
-  faustlive = callPackage ../applications/audio/faust/faustlive.nix { };
+  faustlive = callPackage ../applications/audio/faust/faustlive.nix {
+    llvm = llvm_10;
```

Comment:
any reason not to just depend on `llvm_10` in `faustlive.nix`?

Reply:
I hardly use FaustLive, do you still think I should add myself as a maintainer?
@pmahoney cleaned up the faust package I made a few years ago, but has hardly been involved since.

Reply:
It's preferred that all packages have a maintainer set.

@NegaNexus seems to have been the person who added it (in #45493); @NegaNexus, would you be interested in being set as the maintainer?

Reply:
I've added myself as a maintainer.
@NegaNexus, if you'd like to take over, or add yourself, please do!